content/hi/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,649 @@
+>}}), फिर [नोड नियंत्रक](#नोड-नियंत्रक) ट्रिगर करता है
+{{< glossary_tooltip text="एपीआई-इनीशिएटेड एविक्शन" term_id="api-eviction" >}}
+उस नोड को सौंपे गए सभी पॉड्स के लिए। डिफ़ॉल्ट निष्कासन समयबाह्य अवधि है
+**पाँच मिनट**।
+कुछ मामलों में जब नोड पहुंच योग्य नहीं होता है, तो एपीआई सर्वर संवाद करने में असमर्थ होता है
+नोड पर क्यूबलेट के साथ। पॉड्स को हटाने के निर्णय के बारे में सूचित नहीं किया जा सकता है
+क्यूबलेट जब तक एपीआई सर्वर के साथ संचार फिर से स्थापित नहीं हो जाता। इस बीच में,
+विलोपन के लिए निर्धारित पॉड्स विभाजित नोड पर चलना जारी रख सकते हैं।
+
+नोड नियंत्रक पॉड को हटाने के लिए बाध्य नहीं करता है जब तक यह पुष्टि नहीं हो जाती कि वे बंद हो गए हैं
+क्लस्टर में चल रहा है। आप उन पॉड्स को देख सकते हैं जो अगम्य नोड पर चल रहे हैं
+`टर्मिनेटिंग` या `अज्ञात` स्थिति में होना। ऐसे मामलों में जहां कुबेरनेट्स से निष्कर्ष नहीं निकाला जा सकता है
+अंतर्निहित आधारभूत संरचना यदि कोई नोड स्थायी रूप से क्लस्टर छोड़ देता है, तो क्लस्टर व्यवस्थापक
+नोड ऑब्जेक्ट को हाथ से हटाने की आवश्यकता हो सकती है। कुबेरनेट्स से नोड वस्तु को हटाने का कारण बनता है
+नोड पर चल रहे सभी पॉड ऑब्जेक्ट्स को एपीआई सर्वर से हटा दिया जाएगा और उनके नाम को मुक्त कर दिया जाएगा।
+
+जब नोड्स पर समस्याएँ होती हैं, तो कुबेरनेट्स नियंत्रण विमान स्वचालित रूप से बनाता है
+[दाद](/docs/concepts/scheduling-eviction/taint-and-toleration/) जो शर्तों से मेल खाते हैं
+नोड को प्रभावित करना।
+नोड को पॉड असाइन करते समय शेड्यूलर नोड के दोष को ध्यान में रखता है।
+पॉड में {{< glossary_tooltip text="टॉलरेशन्स" term_id="toleration" >}} भी हो सकता है जो
+वे एक नोड पर चलते हैं भले ही इसमें एक विशिष्ट दोष हो।
+
+देखें [शर्त के अनुसार दागी नोड्स](/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-nodes-by-condition) अधिक जानकारी के लिए।
+
+### क्षमता और आवंटन योग्य {#क्षमता}
+
+नोड पर उपलब्ध संसाधनों का वर्णन करता है: सीपीयू, मेमोरी और अधिकतम
+पॉड्स की संख्या जिन्हें नोड पर शेड्यूल किया जा सकता है।
+
+क्षमता ब्लॉक में फ़ील्ड संसाधनों की कुल मात्रा को इंगित करते हैं जो a
+नोड है। आवंटन योग्य ब्लॉक संसाधनों की मात्रा को इंगित करता है
+नोड जो सामान्य पॉड्स द्वारा उपभोग के लिए उपलब्ध है।
+
+सीखने के दौरान आप क्षमता और आवंटन योग्य संसाधनों के बारे में अधिक पढ़ सकते हैं
+[संसाधनों को आरक्षित करें](/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable)
+एक नोड पर।
+
+### जानकारी
+
+नोड के बारे में सामान्य जानकारी का वर्णन करता है, जैसे कर्नेल संस्करण, कुबेरनेट्स
+संस्करण (क्यूबलेट और क्यूब-प्रॉक्सी संस्करण), कंटेनर रनटाइम विवरण, और कौन सा
+ऑपरेटिंग सिस्टम नोड उपयोग करता है।
+क्यूबलेट इस जानकारी को नोड से इकट्ठा करता है और इसमें प्रकाशित करता है
+कुबेरनेट्स एपीआई।
+
+## दिल की धडकने
+
+कुबेरनेट्स नोड्स द्वारा भेजी गई दिल की धड़कन, आपके क्लस्टर को यह निर्धारित करने में मदद करती है
+प्रत्येक नोड की उपलब्धता, और विफलताओं का पता चलने पर कार्रवाई करना।
+
+नोड्स के लिए दिल की धड़कन के दो रूप होते हैं:
+
+* नोड के `.status` के अपडेट
+* [लीज](/docs/concepts/architecture/leases/) ऑब्जेक्ट्स
+  'क्यूब-नोड-लीज' के भीतर
+  {{< glossary_tooltip term_id="नेमस्पेस" text="namespace">}}.
```

Comment:
The values of `term_id` and `text` parameters seem to be swapped. Please ensure that term_id is in English and text is in Hindi.

```suggestion
  {{< glossary_tooltip term_id="namespace" text="नेमस्पेस">}}.
```